assets/scss/base/_reset.scss

```diff
@@ -24,6 +24,7 @@
 		font-size: 100%;
 		font: inherit;
 		vertical-align: baseline;
+                width: 100%;

```

Comment:
This change will potentially have a greater impact than just updating the images in posts as it spans across a lot of high level elements.

Rather than editing the base styles, it would be better to find a more specific style set to amend.

Also, `Reset.scss` should only be updated to newer versions of that library itself.

Reply:
it can be fixed by adding 

`img{max-width:100%;}`